### PR TITLE
[ref] Migrate Footer component

### PR DESCRIPTION
**Pull request by Diegiwg**
# Description

This PR migrates the `Footer` component from Chakra UI to Tailwind CSS and Daisy UI, maintaining the original structure and visual design.

## Changes

- [x] Removed all Chakra UI components from the `Footer`
- [x] Maintained the original structure while adapting to Tailwind CSS and Daisy UI

## Preview

> print

## Issues

- **Epic:** #164
  - [x] **Closes:** #187

### CODE REVIEW

File: packages/components/Footer.tsx [hunk 1]
```diff
@@ -1,64 +1,51 @@
...
+      </div>
     </Section>
   );
 }
 
 function PodCodarLogo() {
   return (
-    <Stack>
+    <div className="flex flex-col items-center gap-2">
       <Logo size="large" />
```
Comment (marco-souza) [inline packages/components/Footer.tsx:25]: question: can you confirm if we have a task to migrate the logo component?

File: packages/components/Footer.tsx [hunk 2]
```diff
@@ -1,64 +1,51 @@
...
+import { useI18n } from "@packages/features/i18n-context";
 
 import Section from "./Section";
 import SocialIconLinks from "./SocialIconLinks";
 
 export default function Footer() {
   return (
     <Section className="bg-neutral-950 px-6 py-6 items-center">
-      <Grid gridTemplateColumns={{ base: "1fr", sm: "repeat(3, 1fr)" }} color="gray.50" css={centerGridCells}>
+      <div className="grid grid-cols-3 text-gray-50">
```
Comment (marco-souza) [inline packages/components/Footer.tsx:13]: suggestion: take a look at tailwind css primitives
- https://tailwindcss.com/docs/responsive-design

```suggestion
      <div className="grid grid-cols-1 sm:grid-cols-3 text-gray-50">
```

File: packages/components/Footer.tsx [hunk 3]
```diff
@@ -1,64 +1,51 @@
...
-      <Text>{t("contribution")}</Text>
-      <Text fontSize="sm" color="#718096">
-        {PIX_KEY}
-      </Text>
-      <Image src={images.pixQRCode} width="40%" alt={t("contribution")} />
-    </Stack>
+    <div className="flex flex-col items-center gap-2">
+      <p>{t("contribution")}</p>
+      <p className="text-sm text-zinc-500">{PIX_KEY}</p>
+      <img src={images.pixQRCode} width="40%" alt={t("contribution")} />
```
Comment (marco-souza) [inline packages/components/Footer.tsx:36]: I think we need to use `<Image>` - https://nextjs.org/docs/pages/api-reference/components/image

**Review by marco-souza — CHANGES_REQUESTED** [review]

File: packages/components/Footer.tsx (hunk 1 above)
Comment (Diegiwg) [reply packages/components/Footer.tsx:25]: The `Logo` component does not need to be migrated, it is pure HTML + React

File: packages/components/Footer.tsx (hunk 3 above)
Comment (Diegiwg) [reply packages/components/Footer.tsx]: Thanks for the redirection, changed in: https://github.com/podcodar/webapp/pull/189/commits/3511cd519bd0a51b603de256eab747335ec798ad

**Review by marco-souza — APPROVED** [review]
LGTM

**Pull request merged**